com.unity.ml-agents.extensions/Runtime/Sensors/GridSensor.cs

```diff
@@ -7,16 +8,21 @@
 
 namespace Unity.MLAgents.Extensions.Sensors
 {
+    /// <summary>
+    /// Enum describing what kind of depth type the data should be organized as
+    /// </summary>
+    public enum GridDepthType { Channel, ChannelHot, Counting };
+
     /// <summary>
     /// Grid-based sensor.
     /// </summary>
     [AddComponentMenu("ML Agents/Grid Sensor", (int)MenuGroup.Sensors)]
```

Comment:
this should move to the component class.

Reply:
moved it there and forgot to remove here, thanks